src/Orchard.Web/Modules/Orchard.Blogs/AdminMenu.cs

```diff
@@ -26,7 +26,7 @@ public class AdminMenu : INavigationProvider {
         }
 
         private void BuildMenu(NavigationItemBuilder menu) {
-            var blogs = _blogService.Get().Where(x => _authorizationService.TryCheckAccess(Permissions.MetaListBlogs, _workContextAccessor.GetContext().CurrentUser, x)).ToArray();
+            var blogs = _blogService.Get().Where(x => _authorizationService.TryCheckAccess(Permissions.MetaListOwnBlogs, _workContextAccessor.GetContext().CurrentUser, x)).ToArray();
```

Comment:
This change is not necessary, there is an event handler that will do the conversion automatically: `BlogAuthorizationEventHandler`